presto-accumulo/src/main/java/com/facebook/presto/accumulo/AccumuloConnector.java

```diff
@@ -94,9 +96,13 @@ public ConnectorTransactionHandle beginTransaction(IsolationLevel isolationLevel
     }
 
     @Override
-    public void commit(ConnectorTransactionHandle transactionHandle)
+    public Optional<ConnectorCommitResult> commit(ConnectorTransactionHandle transactionHandle)
```

Comment:
We don't have to use Optional. Let's see if we can have `public static final EMPTY_HANDLE = new HiveCommitHandle();` in it.